frontend/src/Utilities/Api.ts

```diff
@@ -54,20 +55,21 @@ export const getTransactionListChunk = async (
     .execute();
 };
 
-export const createAccount = async (
-  accountName: string,
-  initialBalance: number
-) => {
-  return "RESPONSE";
+export const postAccount = async (data: IAccount) => {
```

Comment:
Matau, kad kitos API funkcijos irgi blogai pavadintos, bet man atrodo geriau palikti pavadinta `createAccount`, o ne `postAccount`. Visu pirma, aisku ka daro. Visu antra, kai bus antras post requestas i accountus, pvz. kai jau turima accounta updatinsim su naujais duomenimis, ar tai irgi bus pavadinta `postAccount2`? :D

Reply:
Šiaip standartiškai postAccount būtų su intensija 'Sukurk jei nėra, paupdate'ink jei yra', not sure ar čia pas mus taip vyksta

Reply:
Hmm maybe